imu_filter_madgwick/CMakeLists.txt

```diff
@@ -24,12 +24,13 @@ include_directories(
 
 
 # create imu_filter library
-add_library (imu_filter src/imu_filter.cpp  src/imu_filter_ros.cpp  include/imu_filter_madgwick/imu_filter.h)
+set(CMAKE_CXX_FLAGS "-O2")
```

Comment:
You shouldn't override `CMAKE_CXX_FLAGS`. First, the CMakeLists.txt is the wrong place to do this; if you want a release build, do something like `catkin_make -DCMAKE_BUILD_TYPE=Release` instead. Second, the original contents of `CMAKE_CXX_FLAGS` are lost, which can break the whole catkin workspace or break the build on different architectures. Just remove that line.